apollo-router/src/uplink/mod.rs

```diff
@@ -264,68 +300,100 @@ where
     ReceiverStream::new(receiver).boxed()
 }
 
-pub(crate) async fn fetch<Query, Response>(
+pub(crate) async fn fetch<Query, Response, SomethingElse>(
```

Comment:
Could we have a more explicit name than `SomethingElse` ? :

Reply:
Sure, I thought it was a cute name when @o0Ignition0o wrote it but in retrospect it was probably a placeholder :) `TransformedResponse`.